<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bgWidget</class>
 <widget class="QWidget" name="bgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName=&quot;bgWidget&quot;] 
{
background-image: url(:/Images/Media/lobby.png);
}

QLabel
{
	color:rgb(255,255,255);
}

QTreeWidget
{
	background-color: rgba(182, 16, 27,100);
	font: 12pt &quot;新宋体&quot;;
	color:rgb(255,255,255);
}

QScrollBar
{
	background-color: rgba(182, 16, 27,255);
}

QGraphicsView
{
	background-color: rgba(182, 16, 27,100);
}

QPushButton[objectName=&quot;btn_back&quot;] 
{
border-image: url(:/Images/Media/btn_bg)0 312 0 0;
color: rgb(255, 255, 255);
}
QPushButton[objectName=&quot;btn_back&quot;]::hover
{ 
border-image: url(:/Images/Media/btn_bg)0 156 0 156;
color: rgb(255, 255, 255);
}
QPushButton[objectName=&quot;btn_back&quot;]::pressed
{ 
border-image: url(:/Images/Media/btn_bg)0 0 0 312;
color: rgb(255, 255, 255);
}

QPushButton[objectName=&quot;btn_quit&quot;] 
{
border-image: url(:/Images/Media/btn_bg)0 312 0 0;
color: rgb(255, 255, 255);
}
QPushButton[objectName=&quot;btn_quit&quot;]::hover
{ 
border-image: url(:/Images/Media/btn_bg)0 156 0 156;
color: rgb(255, 255, 255);
}
QPushButton[objectName=&quot;btn_quit&quot;]::pressed
{ 
border-image: url(:/Images/Media/btn_bg)0 0 0 312;
color: rgb(255, 255, 255);
}

QPushButton[objectName=&quot;btn_quickJoin&quot;] 
{
border-image: url(:/Images/Media/btn_quickJoin)0 140 0 0;
}
QPushButton[objectName=&quot;btn_quickJoin&quot;]::hover
{ 
border-image: url(:/Images/Media/btn_quickJoin)0 70 0 70;
}
QPushButton[objectName=&quot;btn_quickJoin&quot;]::pressed
{ 
border-image: url(:/Images/Media/btn_quickJoin)0 0 0 140;
}</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>35</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerImage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="client.qrc">:/Images/Media/nobodyImage.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="nickName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="level">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="goldCoin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>金币：</string>
   </property>
  </widget>
  <widget class="QLabel" name="nickNameText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="levelText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="goldCoinText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quickJoin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>快速加入</string>
   </property>
  </widget>
  <widget class="QLabel" name="curPos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>现在位置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="curPosText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>一区</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="tableList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>731</width>
     <height>551</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTreeWidget" name="gameList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>211</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
